py-tests/pyspark_tests.py

```diff
@@ -0,0 +1,642 @@
+from __future__ import print_function
+import pytest
+import sys
+from operator import add
+import findspark
+findspark.init()
+from pyspark import SparkContext, SparkConf, SQLContext, Row
+import docker, os, subprocess, json, riak, time
+import pyspark_riak
+import timeout_decorator
+import datetime
+#sudo route add 172.17.0.0/16 192.168.99.100, osx only
```

Comment:
The purpose of these comment lines is unclear. If this is some steps that are used in similar manual testing, having them handy here for automating the same steps is helpful and more so than pseudocode, BUT a comment above saying so would help, AND a TODO: to remove this would be great.